src/main/java/org/gaul/s3proxy/S3Exception.java

```diff
@@ -50,6 +51,7 @@
                 Map<String, String> elements) {
         super(requireNonNull(message), cause);
         this.error = requireNonNull(error);
+        this.message = message;
```

Comment:
Can you retrieve this via `super.getMessage`?

Reply:
Done